processor/test/integration/test_projects.js

```diff
@@ -8,7 +8,7 @@ const exec = util.promisify(cp.exec)
 
 const { createApp } = require('../../src/app')
 
-const tmpDir = '/tmp/kitspace-processor-test-from-folder'
+const tmpDir = '/data/test/temp/kitspace-processor-test-from-folder'
```

Comment:
Will this path be treated as an actual temp path?

Reply:
Kind of, I added a `posttest` script that removes the whole `/data/test` directory.